unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/HUDController.cs

```diff
@@ -206,7 +208,7 @@ private void AllUIHiddenOnOnChange(bool current, bool previous)
                     settingsPanelHud.Initialize();
                 break;
             case HUDElementID.AIRDROPPING:
-                CreateHudElement(configuration, hudElementId);
+                await CreateHudElement(configuration, hudElementId);
```

Comment:
I didn't the principle of `awaiting` `CreateHudElement` or not. can you please review the case? In many of them the HUD that should be instantiated is used right after the method but it is not awaited.

Reply:
I added `await` only for the HUD elements that were moved to Addressables. I left other without it just to keep track of the remained, that needed to be refactored. But good point, It is extracted to [this ticket](https://app.zenhub.com/workspaces/workstream-core-experience-63aaac346e1c8d00104586e8/issues/gh/decentraland/unity-renderer/4280) and I mention it

Reply:
Thanks for adding `await` everywhere